test-suite/squidconf/time_units

```diff
@@ -14,35 +14,35 @@
 dns_timeout 1 millisecond
 dns_timeout 0.001 second
 
-# maximum checks
-dns_timeout 9223372036854 milliseconds
-dns_timeout 9223372036 seconds
-dns_timeout 153722867 minutes
-dns_timeout 2562047 hours
-dns_timeout 106751 days
-dns_timeout 15250 weeks
-dns_timeout 7625 fortnights
-dns_timeout 3558 months
-dns_timeout 292.27 years
-dns_timeout 29.227 decades
-
-
```

Comment:
If you have to modify this file for another reason, please undo this whitespace change. Otherwise, I can do it for you during the next review round.

Reply:
> Whitespace changes are for perceived consistency

With the first whitespace change, you may have misread the diff or the author's intent -- those empty lines should be there (well, at least one of them) because they separate test sections from each other.

With the second whitespace change (at EOF), the situation is less certain because some test files use a trailing new line and some do not. Since most Squid sources do use a trailing new line (enforced by `formater.pl` IIRC) _and_ adjusting this whitespace is clearly outside this PR scope, I removed that change too.